package.yaml

```diff
@@ -57,6 +57,11 @@ executables:
     source-dirs: examples
     dependencies: stratosphere
     ghc-options: -Wall
+  api-gateway-lambda:
```

Comment:
I think we need a new system for examples instead of making executables for them. I don't want people to have to compile the examples. Maybe I'll make examples their own package and not upload them to Hackage. Again, I'll make a note, no need to change your PR.